### PR TITLE
e2e/Cypress:fixed spec files

### PR DESCRIPTION
**Pull request by furqanmlk**
#### Summary
channel_info_rhs_spec.js ```fixed```
channel_members_rhs_spec.js ```fixed```
guest_popover_ui_spec.ts ```demoted```
image_link_preview_spec.js ```fixed```
mage_attachment_spec.js ```fixed```

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Comment by furqanmlk** [comment]
e2e-test

File: e2e/cypress/tests/integration/channel/channel_members_rhs_spec.js [hunk 1]
```diff
@@ -155,13 +155,13 @@ describe('Channel members RHS', () => {
                 // # Search for first user
                 cy.uiGetRHS().findByTestId('channel-member-rhs-search').should('be.visible').type(users[0].username);
 
-                // * we should see them, but nobody else
-                cy.uiGetRHS().findByText(`${users[0].username}`).should('be.visible');
+                // // * we should see them, but nobody else
```
Comment (yasserfaraazkhan) [inline e2e/cypress/tests/integration/channel/channel_members_rhs_spec.js:158]: extra `//` here

**Review by yasserfaraazkhan — APPROVED** [review]
great!

**Comment by furqanmlk** [comment]
e2e-test

**Comment by furqanmlk** [comment]
e2e-test

**Pull request merged**